src/types/options/index.ts

```diff
@@ -70,28 +70,28 @@ export type OptionsSource = 'interactive_message' | 'dialog_suggestion' | 'block
  * Type function which given an options source `Source` returns a corresponding type for the `ack()` function. The
  * function is used to fulfill the options request from a listener or middleware.
  */
-type OptionsAckFn<Source extends OptionsSource> = Source extends 'block_suggestion'
+export type OptionsAckFn<Source extends OptionsSource> = Source extends 'block_suggestion'
```

Comment:
As we don't expose other `XXXAckFn` (specifically `ActionAckFn` and `ViewAckFn`) at this point and perhaps you don't need this specific details, can you keep this one as-is?

```suggestion
type OptionsAckFn<Source extends OptionsSource> = Source extends 'block_suggestion'
```

Reply:
Yeah I can leave that one out since it is not needed for development